ant-networking/src/cmd.rs

```diff
@@ -103,7 +103,7 @@ pub enum LocalSwarmCmd {
     /// Returns the quoting metrics and whether the record at `key` is already stored locally
     GetLocalQuotingMetrics {
         key: RecordKey,
-        sender: oneshot::Sender<(QuotingMetrics, bool)>,
+        sender: oneshot::Sender<Option<(QuotingMetrics, bool)>>,
```

Comment:
Why return an option here?

Reply:
Lemme just try to return an error instead of None type there. That makes more sense and it is cleaner